### PR TITLE
📝 improve installation guide

### PR DESCRIPTION
**Pull request by manuel-rw**
*Thank you for contributing to Homarr! So that your Pull Request can be handled effectively, please populate the following fields (delete sections that are not applicable)*

### Category
Documentation

### Overview
Facial uplift of installation guide, improvements on text, update broken links

### CODE REVIEW

File: docs/introduction/installation.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+To add the Homarr container via Docker, simply run the following command in your terminal.
+
+```bash
+docker run  \
+  --name homarr \
+  --restart unless-stopped \
+  -p 7575:7575 \
+  -v ./homarr/configs:/app/data/configs \
+  -v ./homarr/icons:/app/public/icons \
```
Comment (ajnart) [inline docs/introduction/installation.md:34]: This command won't work. Docker run needs absolute paths (I've had reports of it but didn't test)

File: docs/introduction/installation.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+After that, run the same command as before when you installed Homarr.
+
+This process can get tideous, if you update frequently. Thus, we recommend [the installation using docker-compose](#-installation-using-docker-compose-or-portainer) for more experienced users.
+
+## 🐋 Installation using Docker Compose or Portainer
+
+For advanced users, we recommend installation using ``docker-compose`` or [Portainer](https://www.portainer.io/).
+
+```yml title="docker-compose.yml"
+---
```
Comment (ajnart) [inline docs/introduction/installation.md:58]: We can remove this if you want

File: docs/introduction/installation.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      <li>Clone the Repository using <code>git clone https://github.com/ajnart/homarr.git</code></li>
+      <li>Enter the created directory using <code>cd homarr</code></li>
+      <li>Install all dependencies using <code>npm install</code></li>
+      <li>Build the source using <code>npm build</code></li>
+      <li>Start the NextJS web server using <code>npm start</code></li>
+    </ul>
+  </TabItem>
+</Tabs>
+
+## Troubleshoot your Installation
```
Comment (ajnart) [inline docs/introduction/installation.md:141]: Troubleshooting?

File: docs/introduction/manage-services.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+When you change the name of your service, Homarr will automatically search for a matching icon.
+
+<img src={imageAddService} width={500} alt="screen recording of entering the details in the creation modal" />
+
+<br/>
+<br/>
+
+:::info
+
+Homarr will search [dashboard-icons](https://github.com/walkxcode/dashboard-icons) for any matching icons. You can manually change the value in the box or [use custom icons](../advanced-configuration/custom-icons.md).
```
Comment (ajnart) [inline docs/introduction/manage-services.md:39]: Are you using the updated link?

**Review by ajnart — CHANGES_REQUESTED** [review]
Please see my comments. It's 5:50 am 😅
Don't forget to link this with a PR on the dev readme so the links aren't broken

File: docs/introduction/manage-services.md (hunk 4 above)
Comment (manuel-rw) [reply docs/introduction/manage-services.md:39]: Yes, https://github.com/walkxcode/dashboard-icons should be the newest one.

File: docs/introduction/installation.md (hunk 1 above)
Comment (manuel-rw) [reply docs/introduction/installation.md:34]: I quickly tried on my new Raspberry Pi. You're right, relative paths are not support. TIL 😅

**Pull request merged**